### ISSUE TITLE
Add YAML support

### ISSUE THREAD

**Issue by jdigger** [TWCable/grabbit#26]
Some of the configuration options would get cleaner if YAML were supported. (e.g., comments, reference support, reduced verbosity, etc.) Fortunately JSON is a subset of YAML, so existing configs would still work.

**Comment by jdigger** [comment]
In "real" configuration files we've seen that there is a **_LOT_** of duplication, particularly around things like workflow ids. [The reference support](http://www.yaml.org/spec/1.2/spec.html#id2760395) alone would make maintaining config files much cleaner to maintain.

**Comment by sagarsane** [comment]
Will this be a good start @jdigger ?

```yaml
clientNodeType : 'author'
serverUsername : 'admin'
serverPassword : 'admin'
serverHost : 'localhost'
serverPort : 4502
workflowConfigIds : &configs
  - '/configA'
  - '/configB'
  - '/configC'

pathConfigurations :
  - 
    path : '/a/b'
    excludePaths : 
      - 'c'
      - 'd'
    workflowConfigIds : *configs
  - 
    path : '/x/y'
    excludePaths : 
      - 'z'
    workflowConfigIds : *configs
```

**Comment by jdigger** [comment]
Resolved with #83.
